site/pages/_app.tsx

```diff
@@ -45,11 +48,16 @@ function MyApp({ Component, pageProps, router }: AppProps) {
   }, [locale]);
 
   return (
-    <I18nProvider i18n={i18n}>
-      <GridContainer>
-        <Component {...pageProps} />
-      </GridContainer>
-    </I18nProvider>
+    <MoralisProvider
+      appId={process.env.NEXT_PUBLIC_MORALIS_APP_ID as string}
```

Comment:
I need your input:
i've gotten burned in the past by apps with process.env scattered everywhere. It led to bugs when the env vars changed, because it's untyped or type-asserted and TS didn't throw errors.

For this reason I try to export all process env vars from a typed `constants.ts` file. But I get paranoid that this might lead to secrets being exposed in the client? That's more my lack of understanding at how treeshaking/bundling works.

So we can either:
export all process.env vars from `site/lib/constants`
or create a `secrets.ts` and `constants.ts` file and export from both
or keep using process.env

preference?